project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 addSbtPlugin("org.typelevel" % "sbt-typelevel" % "0.4.21")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.4")
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.10.1")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.13.1")
```

Comment:
This is neccessary because the previous version of scala.js was too outdated for Scala 3.3, i.e.

```
[error] org.scalajs.ir.IRVersionNotSupportedException: Failed to deserialize a file compiled with Scala.js 1.12 (supported up to: 1.8): /home/runner/work/jawn/jawn/parser/js/target/scala-3.3.0/test-classes/org/typelevel/jawn/parser/SyntaxCheck$JArray.sjsir
[error] You may need to upgrade the Scala.js sbt plugin to version 1.12 or later.
```